src/runtime/components/forms/Form.vue

```diff
@@ -255,21 +256,18 @@ async function getJoiErrors (
   }
 }
 
-function isValibotSchema (schema: any): schema is ValibotObjectSchema<any> {
-  return schema._parse !== undefined
+function isValibotSchema (schema: any): schema is ValibotSchema | ValibotSchemaAsync {
+  return schema._run !== undefined
 }
 
 async function getValibotError (
   state: any,
-  schema: ValibotObjectSchema<any>
+  schema: ValibotSchema | ValibotSchemaAsync
 ): Promise<FormError[]> {
-  const result = await schema._parse(state)
-  if (result.issues) {
-    return result.issues.map((issue) => ({
-      path: issue.path?.map(p => p.key).join('.') || '',
-      message: issue.message
-    }))
-  }
-  return []
+  const result = await valibotParse(schema, state)
```

Comment:
Is there a way to include the parse function inside the schema object so we can fix issues related to the use of `_parse` without a direct dependency to `valibot`?